src/howitz/endpoints.py

```diff
@@ -518,6 +553,23 @@ def refresh_events():
                            removed_event_list=removed_events, added_event_list=added_events)
 
 
+@main.route('/test_connection')
+def test_conn():
+    is_connection_ok = test_zino_connection()
+    caller_id = request.headers.get('HX-Target', None)
+    if not is_connection_ok:
+        current_app.logger.debug('Connection test failed showing error appbar')
+        return render_template('components/feedback/connection-status-bar/error-appbar-content.html',
+                               error_message="Connection to Zino server is lost")
+    if caller_id == 'connection-error-content':  # If connection should be restored after error
+        current_app.logger.debug('Connection test OK, caller ID %s', caller_id)
+        reconnect_to_zino()
+        return render_template('components/feedback/connection-status-bar/success-appbar-content.html')
+    else:
+        current_app.logger.debug('Connection test OK, caller ID %s', caller_id)
+        return render_template('components/feedback/connection-status-bar/success-appbar-content.html')
+
+
```

Comment:
Polish for later: a link zabbix and similar can visit to check health. Need only return http status codes: 200 or 50x.